.github/workflows/issue_triage.yml

```diff
@@ -0,0 +1,13 @@
+name: Issue Triage
+
+on:
+  issues:
+    types:
+      - opened
+      - unlabeled
+
+jobs:
+  community_issue_triage:
```

Comment:
nit: Our convention in the provider repos looks like to be using dashes in job names.